src/test/java/plugins/AnalysisCollectorPluginTest.java

```diff
@@ -329,6 +330,36 @@ public void should_aggregate_warnings_in_dashboard_portlet() {
         assertThat(dashboard, not(hasWarningsFor(job, TASKS, TASKS_ALL)));
     }
 
+    @Test
+    @WithPlugins({"git@2.4.0", "workflow-aggregator@1.10"})
+    public void should_compute_annotations_on_workflow() {
+        MavenInstallation.installMaven(jenkins, "M3", "3.3.3");
+        WorkflowJob job = jenkins.jobs.create(WorkflowJob.class);
+        // TODO: Job.copyResource(String) does not work for WorkflowJob, since it does not owns
+        // a workspace but the build does (actually it could own 0..N workspaces).
+        job.script.set(
+            "node {\n" +
+            "  git 'https://github.com/amuniz/maven-helloworld.git'\n" +
```

Comment:
Do not like the idea of an automated test relying on some mutable repository in some personal account.